packages/swarm-browser/webpack.config.js

```diff
@@ -0,0 +1,86 @@
+'use strict'
+
+const path = require('path')
+const TerserPlugin = require('terser-webpack-plugin')
+const isProduction = process.env.NODE_ENV === 'production'
+
+const conf = (env, argv) => {
+  const envName = isProduction ? 'production' : 'development'
+  return {
+    bail: Boolean(isProduction),
```

Comment:
This `Boolean()` cast seems unnecessary, looks like `isProduction` is a boolean anyways.